.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,95 @@
+# SPDX-FileCopyrightText: Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Feature Request Form
+description: Request new or improved functionality or changes to existing cuSpatial functionality
+title: "[FEA]: "
+labels: ["feature request", "? - Needs Triage"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to help cuSpatial and fill out this feature request!
+
+  - type: dropdown
+    id: new_or_improvement
+    attributes:
+      label: Is this a new feature, an improvement, or a change to existing functionality?
+      options:
+        - New Feature
+        - Improvement
+        - Change
+    validations:
+      required: true
+
+  - type: dropdown
+    id: criticality
+    attributes:
+      label: How would you describe the priority of this feature request
+      options:
+        - Critical (currently preventing usage)
+        - High
+        - Medium
+        - Low (would be nice)
+    validations:
+      required: true
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: Please provide a clear description of problem this feature solves
+      description: Real usage examples are especially helpful, non-code.
+      placeholder: I want cuSpatial to do _____, because I need to _____.
+    validations:
+      required: true
+
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe your ideal solution
+      description: Please describe the functionality you would like added.
+      placeholder: >
+        A new function that takes in the information in this form, and triages the issue
+
+        def feature_request(form_info):
+            parse(form_info)
+            return triage_outcome
+    validations:
+      required: true
```

Comment:
I suggest cutting this one. I don't really like explicitly asking the user to describe the solution. Expert users are likely to provide this information anyway. Naive users sometimes describe a solution that may not best suit their problem or our library, or would be inefficient. I would rather have a detailed discussion about the problem and come up with the right way to solve it.

Reply:
Fair, I'm not attached to this, but wanted to let users who felt strongly about an approach lay it out here. They can use the additional context field if they're so inclined though.